data.go

```
@@ -470,7 +497,7 @@ func getApidInstanceInfo() (info apidInstanceInfo, err error) {
 			info.InstanceID = GenerateUUID()
 
 			log.Debugf("Inserting new apid instance id %s", info.InstanceID)
-			db.Exec("INSERT INTO APID (instance_id, apid_cluster_id, last_snapshot_info) VALUES (?,?,?)",
+			_, err = tx.Exec("INSERT INTO APID (instance_id, apid_cluster_id, last_snapshot_info) VALUES (?,?,?)",
```

This err is not checked.

---

Err value is checked. See below...

---

I see, sorry.